ansible/roles/settings/tasks/install.yml

```diff
@@ -6,7 +6,11 @@
   hostname: name={{ hostname }}
 
 - name: Set hostname to /etc/hosts
-  lineinfile: dest={{ hosts_file }} regexp='\d+\.\d+\.\d+\.d+\s+{{ hostname }}$' line="{{ ansible_default_ipv4.address }} {{ hostname }}" state=present
+  lineinfile:
+    path: "{{ hosts_file }}"
```

Comment:
`Path` can be used from ansible 2.3, which puts an unnecessary dependency here. I'm going to use `dest` here.